examples/cross_chain_test/lib.rs

```diff
@@ -0,0 +1,46 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[ink::contract]
+mod cross_chain_test {
+
+    #[ink(storage)]
+    #[derive(Default)]
+    pub struct CrossChainTest {}
+
+    impl CrossChainTest {
+        #[ink(constructor)]
+        pub fn new() -> Self {
+            Self {}
+        }
+
+        #[ink(message)]
+        pub fn call(&mut self, address: AccountId, selector: [u8; 4]) {
+            use ink::env::{
+                call::{
+                    build_call,
+                    Call,
+                    ExecutionInput,
+                    Selector,
+                },
+                DefaultEnvironment,
+            };
+
+            let result = build_call::<DefaultEnvironment>()
+                .call_type(Call::new().callee(address))
+                .exec_input(ExecutionInput::new(Selector::new(selector)))
+                .returns::<Result<(), ::ink::LangError>>()
```

Comment:
If messages are always expected to return `Result<T, LangError>`, this should be made implicit in the call builder.

Reply:
Yeah I was thinking of doing that in a follow up PR since it's a source-code level breaking change, not an ABI one